notebooks/summit/AuxTel/SITCOM-802/SITCOM-790_Sky_Flat_Sequence.ipynb

```diff
@@ -0,0 +1,1677 @@
+{
```

Comment:
<p>In my experience, once you are past the saturation point as long as you take the images in succession you can usually get away with using the same exptime for a few minutes at a time without the counts dropping below you target value (10k in this case). So instead of incrementing it each loop, I think we should try to setup a way to only change the exposure time once the counts drop below a specific value.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>